content/pipes.md

```diff
@@ -295,6 +295,16 @@ In this case, we want to bind the pipe at the method call level. In our current
 2. Pass the context-specific Joi schema in the class constructor of the pipe
 3. Bind the pipe to the method
 
+Joi schema example:
```

Comment:
could you add the typescript definition of that `CreateCatDto` as well?

it could leave in the same snippet as the `createCatSchema` one

Reply:
@micalevisk done.

Reply:
you can click on 'resolve conversation' buton then :)

Reply:
@micalevisk The "resolve conversation" button is not visible. Can you resolve it for us?

Reply:
@jared-leddy  It isn't visible for me, that's why I asked him xD Only Kamil and the author (@carbondigitalus) can do that, I guess